src/dev-app/dev-app-chrome/dev-app-chrome-module.ts

```diff
@@ -0,0 +1,27 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {
+  MatButtonModule,
+  MatIconModule,
+  MatListModule,
+  MatSidenavModule,
+  MatToolbarModule
+} from '@angular/material';
+import {RouterModule} from '@angular/router';
+import {DevAppChrome} from './dev-app-chrome';
```

Comment:
Putting `chrome` in the name makes it sound like it has something to do with the browser. Maybe we should call it `DevAppLayout`?